frontend/src/views/HomeView.vue

```diff
@@ -88,13 +97,16 @@ export default {
     }
   },
   computed: {
-    ...mapState(useAuthStore, ['userInfo']),
+    ...mapState(useAuthStore, ['userInfo', 'isMinistryUser', 'impersonateId', 'permissions']),
     isLoading() {
       return this.loading || this.loadingFA
     },
     showReportsAlertBanner() {
       return this.hasPermission(this.PERMISSIONS.SUBMIT_DRAFT_REPORTS) && this.pendingReportsCount > 0
     },
+    isMinistryUserRequiredToImpersonate() {
+      return this.isMinistryUser && !this.impersonatedId && (!Array.isArray(this.permissions) || this.permissions.length === 0)
```

Comment:
There's a typo on **impersonatedId** that is throwing console errors. Also, was there any reason to add a permissions check? If the screen is empty because they are impersonating a user with no permissions I don't think we need to show the alert. We have a separate ticket to prevent users from impersonating an account with no permissions.

Reply:
Oops,  thanks for the good catch on type, and I totally agree with no permission check.    Commit is done accordingly.